sourcegraph/browser-extensions
# fix: missing configure Sourcegraph buttons with empty URL

This PR resolves https://github.com/sourcegraph/sourcegraph/issues/12930:

- Checks for a Sourcegraph URL before fetching to fix requests going to GitHub.com.
- Removes unnecessary safari check (safari is not currently maintained)
- Renders **Configure Sourcegraph** button in page headers when no URL is set or only https://sourcegraph.com is set and user is viewing a private repository.
- Clicking **Configure Sourcegraph** with options page feature flag enabled opens new options page in a new tab.

Testing plan:

- Remove Sourcegraph URL from the feature flagged options page.
- Go to a private repository
- **Configure Sourcegraph** button appears in page headers
- Clicking on Configure Sourcegraph goes to new options page with correct error message.

I have tested on:
